tests/test_inscopiximagingextractor.py

```diff
@@ -0,0 +1,52 @@
+import sys
+
+import numpy as np
+import pytest
+from numpy import dtype
+from numpy.testing import assert_array_equal
+
+from roiextractors import InscopixImagingExtractor
+
+from .setup_paths import OPHYS_DATA_PATH
+
+
+@pytest.mark.skipif(sys.version_info <= (3, 8), reason="Requires Python 3.9 or higher")
+def test_inscopiximagingextractor_movie_128x128x100_part1():
+    file_path = OPHYS_DATA_PATH / "imaging_datasets" / "inscopix" / "movie_128x128x100_part1.isxd"
+    extractor = InscopixImagingExtractor(file_path=file_path)
+
+    assert extractor.get_num_frames() == 100
+    assert extractor.get_image_size() == (128, 128)
+    assert extractor.get_dtype() == dtype("float32")
```

Comment:
This is geting the method from the base class which makes a call to a read operation to get the dtype. 

To avoid this read operation everytime that someeone calls dtype maybe we can determine this at `__init__`

I was hoping that this information was available on the footer and there is a dtype but I am not sure how to map it to numpy dtypes. Check out the footer for this test:

```
print(extractor.movie.footer)
{'dataType': 1, 'extraProperties': None, 'fileVersion': 1, 'hasFrameHeaderFooter': False, 'producer': {'name': 'isxcore', 'version': [1, 8, 0]}, 'spacingInfo': {'numPixels': {'x': 128, 'y': 128}, 'pixelSize': {'x': {'den': 1, 'num': 3}, 'y': {'den': 1, 'num': 3}}, 'topLeft': {'x': {'den': 1, 'num': 0}, 'y': {'den': 1, 'num': 0}}}, 'timingInfo': {'blank': [], 'cropped': [], 'dropped': [], 'numTimes': 100, 'period': {'den': 1000, 'num': 100}, 'start': {'secsSinceEpoch': {'den': 1, 'num': 0}, 'utcOffset': 0}}, 'type': 0}
```

Reply:
Ah, this can be extracted from `self.movie.data_type` see:

https://inscopix.github.io/py_isx/how-to/read-movie-data-type.html